site/themes/carvel/layouts/shortcodes/table.html

```diff
@@ -2,9 +2,19 @@
 # Original table shortcode
 # https://github.com/kubernetes/website/blob/master/layouts/shortcodes/table.html
 -->
-{{ $hasCaption := isset .Params "caption" }}
-{{ $caption    := .Get "caption" }}
-{{ $captionEl  := printf "<table><caption style=\"display: none;\">%s</caption>" $caption }}
-{{ $table      := .Inner | markdownify }}
-{{ $html       := cond $hasCaption ($table | replaceRE "<table>" $captionEl) $table | safeHTML }}
-{{ $html }}
+{{ $table := .Inner | markdownify }}
+
+{{ $table := replace $table "(tr)" "<tr >" }}
+{{ $table := replace $table "(/tr)" "</tr>" }}
+{{ $table := replace $table "(th)" "<th>" }}
+{{ $table := replace $table "(/th)" "</th>" }}
+{{ $table := replace $table "(br)" "<br />" }}
+{{ $table := replace $table "(td)" "<td nowrap>" }} 
```

Comment:
If we could push the `nowrap` effect into CSS, then this shortcode becomes fully reusable.

Look into https://www.w3schools.com/cssref/pr_text_white-space.asp

Reply:
Great suggestions! thanks John